FireRender.Maya.Src/scripts/AERPRVolumeTemplate.mel

```diff
@@ -150,10 +150,14 @@ global proc albedoGridSelectReplace( string $attrName, string $listAttrName )
 	string $multiAttr[] = `listAttr -multi ( $listAttrName )`;
 	for ( $attr in $multiAttr )
 	{
-		$len = size($listAttrName) - size($attr) + 3; // 3 is length of "[0]"
+		string $buffer[];
```

Comment:
Exactly same code in 3 functions: albedoreplace ,dencity replace etc ?
Could you please refactor in order to prevent code duplication ?